dist/s_clang_scan

```diff
@@ -39,26 +39,21 @@ rm -rf ${t_build} && mkdir ${t_build}
 cd ${t_build}
 args="-o clangScanBuildReports"
 args="$args --use-cc=$compiler"
-args="$args -disable-checker core.NullDereference"
```

Comment:
I will probably need to re-enable this.

Reply:
Saw some related dump in the PR build

Reply:
Yes, I now understand why it was disabled.